init-config.sh

```diff
@@ -35,6 +44,12 @@ touch zabbix/odbc.ini
 echo "alpine-latest" >zabbix/container.version
 
 echo "ZBX_HOSTNAME=${HOSTNAME}" >env.list
-echo "ZBX_SERVER_HOST=${SERVER_HOST}" >>env.list
+if [ -z "$PASSIVE_PROXY" ]; then
+  echo "ZBX_SERVER_HOST=${SERVER_HOST}" >>env.list
+else
+  echo "ZBX_PROXYMODE=1" >>env.list
+fi
 echo "ZBX_CONFIGFREQUENCY=300" >>env.list
 echo "ZBX_CACHESIZE=120M" >>env.list
+echo "ZBX_STARTHTTPPOLLERS=10" >>env.list
```

Comment:
This seems rather large "default" value.

Reply:
It practically means that Zabbix Proxy launches ten threads ready to serve for http polling requests. In my opinion, ten threads do not count very much in terms of server resources - but that of course depends very much on the target environment. Hmm we could also add interactive question about the size of the target environment (eg. S / M / XL) and scale the parameters automatically.

Reply:
I agree. I'd say that feature can wait until actual need arises.